test/scripts/cloudformation-template.coffee

```diff
@@ -8,7 +8,7 @@ appRoot = "test/data/test-app/"
 env = "test"
 
 call ->
-  config = yield configuration.read appRoot
+  config = yield configuration.readApp appRoot
```

Comment:
I specifically wanted configurable approot so I could point this at various test setups.